### PR TITLE
Update developing-with-ce.asciidoc

### PR DESCRIPTION
**Pull request by kaviwe**
@Peppe

### CODE REVIEW

File: articles/ce/developing-with-ce.asciidoc [hunk 1]
```diff
@@ -1,17 +1,49 @@
 ---
-title: Trial and Development Use
+title: Licensing and Pricing Model
 order: 3
 layout: page
 ---
 
 [[ce.developing]]
-= Trial and Development Use
+= Licensing
```
Comment (Peppe) [inline articles/ce/developing-with-ce.asciidoc:8]:
```suggestion
= Licensing and Pricing Model
```
I **think** that the first header should always be the same as the title.

File: articles/ce/developing-with-ce.asciidoc [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
-To get started, follow the <<tutorial#,tutorial on adding Collaboration Engine to your project>> to learn about the features.
+When you take your application to production, you need a valid license file.
+To get started, follow the <<tutorial#,Quick Start Guide>> to learn about adding Collaboration Engine to your Vaadin application, and using its high-level APIs.
 
 While your application is in development mode, you can use all the features of Collaboration Engine without restrictions.
 You do not need a development license or a separate trial.
 Development mode is the default mode of a Vaadin application and is used when not explicitly configured to do a production build.
+
+[[ce.developing.pricing-model]]
+== Billing model
```
Comment (Peppe) [inline articles/ce/developing-with-ce.asciidoc:20]:
```suggestion
== Billing Model
```
Always Title Case Headers

File: articles/ce/developing-with-ce.asciidoc [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
+
+There are two options for taking an application that uses Collaboration Engine into production:
+
+* Free "Universal" License (works with any Vaadin application)
+* Commercial License (requires an active commercial Vaadin subscription)
+
+[[ce.developing.free-universal-license]]
+== Free "Universal" License
+
+The "Universal" License is offered free to all Vaadin developers. It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month. Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.
```
Comment (Peppe) [inline articles/ce/developing-with-ce.asciidoc:32]:
```suggestion
The "Universal" License is offered free to all Vaadin developers.
It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.
```
One sentence per line.

Comment (Peppe) [inline articles/ce/developing-with-ce.asciidoc:32]: > Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.

Some words are missing here. (and text checker did not like the word obtain)

**Review by Peppe — CHANGES_REQUESTED** [review]

File: articles/ce/developing-with-ce.asciidoc (hunk 1 above)
Comment (Peppe) [reply articles/ce/developing-with-ce.asciidoc]: @jouni was there some rule like this?

File: articles/ce/developing-with-ce.asciidoc (hunk 3 above)
Comment (kaviwe) [reply articles/ce/developing-with-ce.asciidoc]: The "Universal" License is free for all Vaadin developers. It allows up to 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month. Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to use this license.

Comment (kaviwe) [reply articles/ce/developing-with-ce.asciidoc]: @Peppe Would this work? Still don't know how to add suggestions... Sorry.

Comment (Peppe) [reply articles/ce/developing-with-ce.asciidoc]: Sounds good. Will edit suggestion.

File: articles/ce/developing-with-ce.asciidoc [hunk 4]
```diff
@@ -1,17 +1,50 @@
...
+
+* Free "Universal" License (works with any Vaadin application)
+* Commercial License (requires an active commercial Vaadin subscription)
+
+[[ce.developing.free-universal-license]]
+== Free "Universal" License
+
+The "Universal" License is offered free to all Vaadin developers.
+It allows 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
+Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to obtain this license.
```
Comment (Peppe) [inline articles/ce/developing-with-ce.asciidoc:34]:
```suggestion
The "Universal" License is free for all Vaadin developers. It allows up to 10 unique end users in a Vaadin application to use features that rely on Collaboration Engine each month.
Developers do not require a commercial Vaadin subscription (Pro, Prime, or Enterprise) to use this license.
```

**Review by Peppe — CHANGES_REQUESTED** [review]

File: articles/ce/developing-with-ce.asciidoc (hunk 3 above)
Comment (Peppe) [reply articles/ce/developing-with-ce.asciidoc]: Added a new edit suggestion.

**Pull request merged**